packages/bootstrap-4/src/EmailWidget/EmailWidget.tsx

```diff
@@ -2,7 +2,7 @@ import React from "react";
 import TextWidget, { TextWidgetProps } from "../TextWidget";
 
 const EmailWidget = (props: TextWidgetProps) => {
-  return <TextWidget type="email" {...props} />;
+  return <TextWidget {...props} type="email" />;
```

Comment:
@Xtremilicious I guess for this to work lint free, I think `type` has to be optional in `index.d.ts` in `widgetProps` .

may be , you can check this

Reply:
Looks like the `type` prop type is being removed. Once that lands, we can sync and merge that change back. This change can then be reverted (or kept, it doesn't matter as the type prop should not be overrideble)

https://github.com/rjsf-team/react-jsonschema-form/pull/1826

Reply:
@anikethsaha should we merge this or wait for now?

Reply:
we have to wait for rjsf-team#1826

Reply:
@anikethsaha I resolved the conflicts, you can now merge 👍